From 9daade0b95bbc5443637977652d3cd0dbc44e112 Mon Sep 17 00:00:00 2001
From: muxator <muxator@users.noreply.github.com>
Date: Fri, 13 Apr 2018 18:32:39 +0200
Subject: [PATCH] fix: line numbers was not aligned with text

This change partially reverts 0a9d02562d7e, which got released in 1.6.4
due to #3280.

Text size and line alignment are now reverted back to their 1.6.3
appearance (thus stay non customizable, for now).

Fixes #3378
---
 src/static/css/iframe_editor.css | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/src/static/css/iframe_editor.css b/src/static/css/iframe_editor.css
index 0286a5df8..9aa003aaf 100644
--- a/src/static/css/iframe_editor.css
+++ b/src/static/css/iframe_editor.css
@@ -31,17 +31,13 @@ body {
 body.grayedout { background-color: #eee !important }
 
 #innerdocbody {
-  font-size: 16px; /* overridden by body.style */
+  font-size: 12px; /* overridden by body.style */
   font-family:Arial, sans-serif; /* overridden by body.style */
-  line-height: 22px; /* overridden by body.style */
+  line-height: 16px; /* overridden by body.style */
   background-color: white;
   color: black;
 }
 
-.innerdocbody>div{
-  padding: 1px;
-}
-
 body.doesWrap {
   /* white-space: pre-wrap; */
 
